src/services/api.js

```diff
@@ -27,3 +27,31 @@ export async function fetchRestaurant({ restaurantId }) {
   const data = await response.json();
   return data;
 }
+
+export async function postLogin({ email, password }) {
+  const url = 'https://eatgo-login-api.ahastudio.com/session';
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email, password }),
+  });
+  const { accessToken } = await response.json();
+  return accessToken;
+}
+
+export async function postReview({
+  accessToken, restaurantId, score, description,
+}) {
+  const url = 'https://eatgo-customer-api.ahastudio.com'
+  + `/restaurants/${restaurantId}/reviews`;
```

Comment:
```suggestion
  const baseURL = 'https://eatgo-customer-api.ahastudio.com';
  
  const url = `${baseURL}/restaurants/${restaurantId}/reviews`;
```

처럼 줄여주는 방법도 있을 것 같아요.

Reply:
더 나아가면  

```javascript
const url = [
  baseURL, 'restaurants', restaurantId,  'reviews'
].join('/');

같이 정리해줄 수도 있겠네요 ;)

코드를 표현하는 방법은 다양하니 코드가 길게 느껴지거나 표현이 어색할 때 리팩터링 고민을 해보는 것도  좋아요.
```

Reply:
네 한번 수정해보았습니다.
이부분이 리팩토링이 필요한가를 곰곰히 생각해보고 적용해보았습니다.
앱이 커질수록 중복되는 부분과 주소가 바뀌게 되었을때의 경우를 생각하였을때 baseURL로 정리해주면 좋을 것 같네요 !!! 감사합니다 👍